scene_exceptions/scene_exceptions_tvdb.json

```diff
@@ -682,6 +682,13 @@
                 "Dragons Den UK"
             ]
         },
+        "79895": {
+            "8": [
+                "Gintama S4",
+                "Gintama.",
```

Comment:
In fact, it's an alias for the show too, and it is used by Deadfish team

https://www.nyaa.se/?page=search&cats=0_0&filter=0&term=Gintama.